src/components/AccessibilityGuide/ProgressiveEnhancement.mdx

```diff
@@ -0,0 +1,109 @@
+- {<Link href="#component-audit" target="_self">Component Audit</Link>}
+- {<Link href="#general-recommendations" target="_self">General Recommendations</Link>}
+- {<Link href="#resources" target="_self">Resources</Link>}
+
+## General Information
+
+Progressive enhancement is a frontend development strategy and mindset that
+emphasizes building a "basic", functional version of a website or component first,
+and then adding more advanced features and enhancements. This approach ensures
+that the core functionality of a component or site is accessible to all users,
+regardless of their device or browser capabilities. Once the necessary functionality
+is in place, we can provide additional enhancements for users with modern browsers
+and devices.
+
+This approach is particularly important for accessibility and users with
+disabilities, who may be using assistive technologies or older devices that do
+not support advanced features. This inclusive design philosophy also focuses on
+developers knowing and implement web fundamentals, such as HTML and CSS, before
+heavily relying on JavaScript.
+
+## NYPL and DS Problem
+
+When Javascript is turned off in a browser, certain Design System (DS) components
+and general user interfaces do not work. There are various reasons why app-level
+features may not work including how the application is architected, how data is
+fetched, and how data is rendered. Many issues, however, point back to DS
+components.
+
+## HTML Button Issue
+
+HTML `<button>`s can work without javascript in the following contexts.
+
+- When used inside a `<form>` element, if it has an attribute of type=”submit”,
+  the button will be clickable.
+- When used inside a `<form>` element, if it has an attribute of type=”reset”,
+  the button will be clickable and clear out input elements.
+
+There are DS components that use the `Button` component, and therefore the
+`<button>` HTML element, and cannot fully function without Javascript. These
+components rely on the button element to open, close, or perform user actions
+that won’t work without javascript.
+
+## Component Audit
+
+As of v3.6.1, the DS contains 25 fully functional components, and 36 components
+that need Javascript to work.
+
+The following (25) components are fully functional without Javascript:
+
+- `Breadcrumbs`, `Card`, `FeaturedContent`, `FieldSet`, `Footer`, `Form`,
+  `FormRow`, `FormField`, `HelperErrorText`, `Hero`, `HorizontalRule`, `Icon`,
+  `Image`, `Label`, `Link`, `List`, `Logo`, `SimpleGrid`, `SkipNavigation`,
+  `StatusBadge`, `StructuredContent`, `StyledList`, `Table`, `Template`, `Text`
+
+The following (16) use the HTML `<button>` element and are not fully functional without
+Javascript:
+
+- `Accordion`, `AlphabetFilter`, `Banner`, `Button`, `ButtonGroup`, `Header`,
+  `Menu`, `Modal`, `MultiSelect`, `MultiSelectGroup`, `NewsletterSignup`,
+  `Notification`, `Pagination`, `SubNav`, `Tabs`, `TagSet`
```

Comment:
Use a separate bullet for each component.

Reply:
Same comment as above - so do we leave these or is there something we can do?